src/win/tty.c

```diff
@@ -112,14 +112,30 @@ static int uv_tty_virtual_offset = -1;
 static int uv_tty_virtual_height = -1;
 static int uv_tty_virtual_width = -1;
 
+/* The console window size
+ * We keep this separate from uv_tty_virtual_*. We use those values to only
+ * handle signalling SIGWINCH
+ */
+
+static HANDLE uv_tty_console_handle = INVALID_HANDLE_VALUE;
```

Comment:
can you prefix these with `uv__` instead? I know src/win is not very consistent in this regard, but let's try to be, for new code.